.github/workflows/models-post-commit.yaml

```diff
@@ -64,12 +62,9 @@ jobs:
         timeout-minutes: ${{ inputs.timeout }}
         uses: ./.github/actions/docker-run
         with:
-          docker_username: ${{ github.actor }}
+          install_wheel: true
           docker_password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I could not move the docker_password parameter to be filled by default since `secrets` are not available inside of Github custom Actions.

Reply:
have we tried using 
secrets: inherit?
https://docs.github.com/en/actions/sharing-automations/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow

Seem strange that a github action can't get access to custom GH actions

Reply:
yes, I have tried and the secrets are inaccessible in the Github actions - see here 

https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions#accessing-your-secrets

Reply:
Yes this is true - the `secrets` context in general is not available in a composite action. You explictly must pass it